### DISCUSSION TITLE
Netdata vs Prometheus? Deep dive performance analysis! | Netdata Office Hours #8

### DISCUSSION THREAD

**Discussion by hugovalente-pm** [netdata/netdata#16304] [Announcements]
<img src="https://github.com/netdata/netdata/assets/82235632/2a8c0c32-badd-4a2c-8e48-110054ba288d" width="40%" height="40%"/>

Join the Netdata Team, on the 2nd of November at 16:00 UTC for the Netdata Office Hours.

Together we'll cover:
* Spotlight
   * Netdata vs Prometheus Performance Analysis
   * Upcoming Netdata Plan changes
* Open slot for questions.
* Questions & feedback raised by the Community

Links to event:
[YouTube](https://www.youtube.com/watch?v=0x1h6BEVNw0)
[LinkedIn](https://www.linkedin.com/video/live/urn:li:ugcPost:7125874094394269696/?actorCompanyId=27035954)

This event will be available to the Community for rewatch.

Hoping to see you there!

**Comment by carasen12** [comment]
http://192.168.7.117:19999/#;theme=slate;help=true

**Comment by hugovalente-pm** [comment]
updating the streaming links: 
YT - https://www.youtube.com/watch?v=0x1h6BEVNw0
LinkedIn - https://www.linkedin.com/video/live/urn:li:ugcPost:7125874094394269696/
